src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidator.java

```diff
@@ -119,7 +120,12 @@ private void handleRestClientException(
                 .filter(it -> it.ocrData != null)
                 .collect(toList());
         if (docsWithOcr.size() > 1) {
-            log.warn("Multiple documents with OCR in envelope. File name: {}", envelope.zipFileName);
+            log.warn(
```

Comment:
do we get alerts for warning ?

Reply:
should it be an error?

Reply:
yes I think as it not something which can be ignored and scanning supplier would need to fix it.

Reply:
if that's the case, maybe it would be better to just reject the envelope?

Reply:
that was the discussion we had and recommended :) we can continue this discussion on slack channel ?